src/java/org/apache/cassandra/service/StorageServiceMBean.java

```diff
@@ -786,6 +787,10 @@ public StageConcurrency(int corePoolSize, int maximumPoolSize)
      */
     public void stopFullQueryLogger();
 
+    public boolean isFullQueryLogEnabled();
+
+    public FullQueryLoggerOptions getFullQueryLoggerOptions();
```

Comment:
this isn't safe for JMX, should be of types primitive, standard java serializable class (not guava versions of collections), or CompositeData; custom classes shouldn't be used in the JMX